src/stream_base.cc

```diff
@@ -274,26 +282,28 @@ int StreamBase::WriteString(const FunctionCallbackInfo<Value>& args) {
     CHECK_EQ(count, 1);
   }
 
-  AllocatedBuffer data;
+  std::unique_ptr<BackingStore> bs;
 
   if (try_write) {
     // Copy partial data
-    data = AllocatedBuffer::AllocateManaged(env, buf.len);
-    memcpy(data.data(), buf.base, buf.len);
+    NoArrayBufferZeroFillScope no_zero_fill_scope(isolate_data);
+    bs = ArrayBuffer::NewBackingStore(isolate, buf.len);
+    memcpy(reinterpret_cast<char*>(bs->Data()), buf.base, buf.len);
     data_size = buf.len;
   } else {
     // Write it
-    data = AllocatedBuffer::AllocateManaged(env, storage_size);
-    data_size = StringBytes::Write(env->isolate(),
-                                   data.data(),
+    NoArrayBufferZeroFillScope no_zero_fill_scope(isolate_data);
```

Comment:
```suggestion
    NoArrayBufferZeroFillScope no_zero_fill_scope(env->isolate_data());
```

and then drop the `isolate_data` variable above, like in the first `NoArrayBufferZeroFillScope` in this PR?